playbooks/roles/ocp-customization/templates/tuned-sysctl-tuning.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: tuned.openshift.io/v1
+kind: Tuned
+metadata:
+  name: tuned-sysctl-settings
+  namespace: openshift-cluster-node-tuning-operator
+spec:
+  profile:
+  - name: tuned-sysctl-settings
+    data: |
+      [main]
+      summary=Custom sysctl settings for OpenShift nodes
+      include=openshift-node
+
+      [sysctl]
+{% for option in sysctl_options %}
+     - {{ option }}
+{% endfor %}
+
+  recommend:
+  - match:
+{% for dict_item in match_array %}
```

Comment:
I was checking a better way to do it since we are copying the list variable match_array as it is from var. Found about `to_nice_yaml` filter to use in jinja.

Reply:
I can check that

Reply:
I was playing around it seems needs to be tuned for indentation. Here is the sample I was trying:
```
- hosts: all
  gather_facts: no
  vars:
    match_array:
      - label: node-role.kubernetes.io/master
      - label: icp4data
        value: database-db2oltp
        type: pod
      - label: disk
        value: ssd
  tasks:
    - name: Ansible Template Example
      template:
        src: t.j2
        dest: t.txt
```
template:
```
match:
{{ match_array | to_nice_yaml(indent=2) | indent(2, true) }}
```
output:
```
match:
  - label: node-role.kubernetes.io/master
  - label: icp4data
    type: pod
    value: database-db2oltp
  - label: disk
    value: ssd
```

Problem here is it won't validate the match_array and just blindly copy it to the template.

Reply:
which should be fine I think for now. I used a similar one and pushed the new patch